### PR TITLE
Create the directory for the stats file if it doesn't exist

### PR DESCRIPTION
**Pull request by ide**
If the directory to your stats file doesn't yet exist, this plugin throws an error. Think it's nicer to create the directiry if needed before writing the stats file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,30 @@
 {
-  "name":        "stats-webpack-plugin",
```
Comment (ide) [inline package.json]: can revert these changes if you prefer.... npm automatically removed the alignment when I ran `npm install --save`

Comment (unindented) [inline package.json]: Yes please, I'm OCD when it comes to aligning dependencies in `package.json`. 😓

If you revert and add the package by hand, please squash with your previous commit, so that we only have one commit in the end.

**Comment by unindented** [comment]
Good catch! And your pull request even includes a new test... You rock! 👏

**Comment by ide** [comment]
Yeah no problem. Updated the package.json, now with aligned fields.

**Comment by unindented** [comment]
Publishing a new version now. Please hold.

**Comment by unindented** [comment]
Version `0.1.1` published with your fix. Thank you for contributing!

**Comment by ide** [comment]
Thanks Daniel!
